packages/next/src/server/require-hook.ts

```diff
@@ -17,13 +17,18 @@ export function addHookAliases(aliases: [string, string][] = []) {
   }
 }
 
+// We need to use `eval` here to avoid webpack from rewriting the `require.resolve` calls
+// as this code could be bundled
+// eslint-disable-next-line no-eval
+const resolve = eval('require').resolve
```

Comment:
This shouldn't be needed as any of these should be marked as external

Reply:
It is actually because require.resolve gets compiled lol